recipes/pypolyagamma-3/meta.yaml

```diff
@@ -1,43 +1,46 @@
 {% set name = "pypolyagamma-3" %}
-{% set version = "0.1.10" %}
+{% set version = "0.1.11" %}
 
 package:
-  name: {{ name }}
-  version: {{ version }}
+    name: {{ name }}
+    version: {{ version }}
 
 source:
-  url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
-  fn: {{ name }}-{{ version }}.tar.gz
-  sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/pypolyagamma/archive/{{ version }}.tar.gz
+    sha256: 6da114c6398081e8c5e5efca958e7c6e163acc7c9d0ef2c7448cbc35c3382b2d
```

Comment:
The sha256 changes if you change the version. `0.1.11` is `6deef1c854ff090fdd2f05a33a84081d052c026dbd18224325e559ee65e255dc`